### DISCUSSION TITLE
Leaving a studio as its owner

### DISCUSSION THREAD

**Discussion by lisa-wolfgang** [ScratchAddons/ScratchAddons#1649] [Help] [answered]
I have a question about the behavior of the "Leave studio" button added by the "Studio manager tools" addon.
Normally, when you're the owner of a studio, it's impossible to leave without deleting the studio itself. If, however, I use the "Leave studio" button, will I successfully be removed from the studio without deleting it? If so, is the studio broken due to having no owner?

**Comment by cobaltt7** [comment]
i think it'd give an error
________________________________

**Comment by WorldLanguages** [comment, marked as answer]
Scratch Addons won't let you, and even if it did, Scratch API would return an error.

**Comment by Illodrus** [comment]
The only way to leave a studio as its owner is to _delete the account that owns the studio_. I do not suggest this method, as it involves deleting your account.
It is possible and can result in studios with no manager and other such scenarios.
The other functions of the studio will work normally and as you would expect.

For example: https://scratch.mit.edu/studios/26973134/curators/ (excuse the name/description)
This studio has no owner and cannot be deleted. It was made as a test, and if you go back to the first activity message: https://scratch.mit.edu/studios/26973134/activity/4/ you will see that it was made by an account that no longer exists.
